pkg/api/graph/types.go

```diff
@@ -222,8 +223,14 @@ func ImageStreamTag(g MutableUniqueGraph, namespace, name, tag string) graph.Nod
 	if len(tag) == 0 {
 		tag = "latest"
 	}
+	var uname UniqueName
+	if strings.Index(name, ":") == -1 && strings.Index(name, "@") == -1 {
```

Comment:
I assume this covers the case where it's *not* an ImageStreamTag or ImageStreamImage? and the `else` covers those 2 kinds